src/index.js

```diff
@@ -66,63 +52,61 @@ class _ {
  *
  * @param {Object} pluginConfig - Plugin configuration. This is NOT optional!
  * @returns {undefined}
  */
-function installRaven(pluginConfig) {
-	const Raven = pluginConfig.ravenClient;
-	if (!Raven) {
-		console.error("Raven client not found.");
+function installSentry(pluginConfig) {
+	const Sentry = pluginConfig.sentryClient;
+	if (!Sentry) {
+		console.error("Sentry client not found.");
 	}
 
 	// Check for local environment
 	const isLocalEnv = process.env.IS_OFFLINE || process.env.IS_LOCAL || !process.env.LAMBDA_TASK_ROOT;
 	if (pluginConfig.filterLocal && isLocalEnv) {
 		// Running locally.
 		console.warn("Sentry disabled in local environment");
-		delete process.env.SENTRY_DSN; // otherwise raven will start reporting nonetheless
+		delete process.env.SENTRY_DSN; // otherwise sentry will start reporting nonetheless
 
-		Raven.config().install();
+		Sentry.init({ dsn: "" });
 
-		ravenInstalled = true;
+		sentryInstalled = true;
 		return;
 	}
 
-	// We're merging the plugin config options with the Raven options. This
-	// allows us to control all aspects of Raven in a single location -
+	// We're merging the plugin config options with the Sentry options. This
+	// allows us to control all aspects of Sentry in a single location -
 	// our plugin configuration.
-	Raven.config(
-		process.env.SENTRY_DSN,
-		_.extend({
-			release: process.env.SENTRY_RELEASE,
-			environment: isLocalEnv ? "Local" : process.env.SENTRY_ENVIRONMENT,
-			tags: {
-				lambda: process.env.AWS_LAMBDA_FUNCTION_NAME,
-				version: process.env.AWS_LAMBDA_FUNCTION_VERSION,
-				memory_size: process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE,
-				log_group: process.env.AWS_LAMBDA_LOG_GROUP_NAME,
-				log_stream: process.env.AWS_LAMBDA_LOG_STREAM_NAME,
-				service_name: process.env.SERVERLESS_SERVICE,
-				stage: process.env.SERVERLESS_STAGE,
-				alias: process.env.SERVERLESS_ALIAS,
-				region: process.env.SERVERLESS_REGION || process.env.AWS_REGION
-			}
-		}, pluginConfig)
-	).install();
+	Sentry.init(
+		_.extend(
+			{
+				dsn: process.env.SENTRY_DSN,
+				release: process.env.SENTRY_RELEASE,
+				environment: isLocalEnv ? "Local" : process.env.SENTRY_ENVIRONMENT,
+				tags: {
+					lambda: process.env.AWS_LAMBDA_FUNCTION_NAME,
+					version: process.env.AWS_LAMBDA_FUNCTION_VERSION,
+					memory_size: process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE,
+					log_group: process.env.AWS_LAMBDA_LOG_GROUP_NAME,
+					log_stream: process.env.AWS_LAMBDA_LOG_STREAM_NAME,
+					service_name: process.env.SERVERLESS_SERVICE,
+					stage: process.env.SERVERLESS_STAGE,
+					alias: process.env.SERVERLESS_ALIAS,
+					region: process.env.SERVERLESS_REGION || process.env.AWS_REGION
+				}
+			},
+			pluginConfig
+		)
+	);
 
-	// Register this instance globally for backward compatibility
-	// with serverless-sentry-plugin 0.2.x/0.3.x
-	global.sls_raven = Raven;
-	ravenInstalled = true;
+	sentryInstalled = true;
 
-	console.log("Raven installed.");
+	console.log("Sentry installed.");
```

Comment:
I'd be more explicit with logs and namespace them. It's not easy for a developer to understand athat this comes from the plugin and not sentry SDK itself.